### PR TITLE
[irods/irods#7618] Update contact page with steps for reporting security vulnerabilities

### PR DESCRIPTION
**Pull request by korydraughn**
Checking rendering now.

### CODE REVIEW

**Comment by korydraughn** [comment]
Here's a rendering of the changes in Google Chrome.

image

**Review by alanking — COMMENTED** [review]
Seems okay to me. Probably should let @trel or comms or whoever take a look?

File: content/pages/contact.html [hunk 1]
```diff
@@ -13,9 +13,16 @@ <h4><i class="fa fa-comments"></i> <b>Contact Us...</b></h4>
...
+
                 <h3>Address</h3>
-                <a title="iRODS - Google Maps" href="https://goo.gl/maps/Ykp2G">100 Europa Dr. Ste 540<br>
-                Chapel Hill, NC 27517</a>
+                <a title="iRODS - Google Maps" href="https://goo.gl/maps/Ykp2G">
+                100 Europa Dr. Ste 540<br>
+                Chapel Hill, NC 27517
+                </a>
+
+                <h3>Reporting a Security Vulnerability</h3>
```
Comment (korydraughn) [inline content/pages/contact.html:23]: Change to "Reporting Security Vulnerabilities".

Comment (korydraughn) [reply content/pages/contact.html]: Done.

**Comment by trel** [comment]
looks good

**Comment by korydraughn** [comment]
Merging.

**Pull request merged**